docs/user-guides/overlay.md

```diff
@@ -0,0 +1,12 @@
+### OpenGL Overlay
```

Comment:
I would call this file `overlay_openGL.md` in case we at some point want to document the overlay usage in general (for which the filename `overlay.md` is more appropriate than for this specific case) :point_up:

Reply:
Ok, will rename it.